visualization/motion_planning_tasks/src/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ target_link_libraries(${MOVEIT_LIB_NAME}
 target_include_directories(${MOVEIT_LIB_NAME}
 	PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
 	PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
+	PRIVATE ${PROJECT_SOURCE_DIR}/../core/include
```

Comment:
Will this still work with build-farm releases?
I believe packages have to build independently from where they are located and whether or not others are compiled together with them.

I would have expected you add a public interface to get the required position information the visualization requires.

Reply:
Hm. You are right, this will not work for release builds on the build farm.
Actually, I wanted to avoid rewriting the code...

Reply:
I looked into this issue some weeks ago and decided to leave it unresolved for that reason...
But I expect it would take you less time to look at the required interface changes ;)